manifest/kubernetes/rhel/ncp-rhel.yaml

```diff
@@ -971,15 +971,6 @@ spec:
           # we move the usr kmod files to this dir temporarily before
           # installing new OVS kmod and/or backing up existing OVS kmod backup
             mountPath: /tmp/nsx_usr_ovs_kmod_backup
-
-
-          # mount to which an OVS kmod file is copied.
-          - name: host-usr-share
-            mountPath: /host/usr/share
-          # mount host's depmod to create module dependencies after creating
-          # new OVS kmod files
-          - name: host-depmod
-            mountPath: /sbin/depmod
 {{else}}
```

Comment:
We can just remove code line #983 - 985, this commented code is not needed for operator. since {{if .UseNsxOvsKmod}} could have condition to check whether use it or not.
{{else}}
...
# mountPath: /tmp/nsx_usr_ovs_kmod_backup